README.md

```diff
@@ -207,14 +221,14 @@ jsons/ - contains all json files. The json files constitute the data you are sen
 ###  Usage
 
 If you want to write your own configurations then include them in the configs/ directory. If you want
-to write your own json then include them in the jsons/ directory. 
+to write your own json then include them in the jsons/ directory.
 
 It is important to note that you do not need specify the full relative paths
 of the config files when running the framework on the command line. The framework
 will search for all config files in the configs/ directory. When specifying the json
 files in the config files, you do not need to specify the full relative paths
 for the json files, just the name of the file. The framework looks automatically
-in the jsons/ directory for all json files. 
+in the jsons/ directory for all json files.
```

Comment:
Sorry, I do not know :)
I have not gone through the analysis dashboard instructions.